exercises/concept/making-the-grade/.meta/src/reference/java/MakingTheGrade.java

```diff
@@ -53,4 +53,32 @@ static List<String> studentRanking(List<Integer> studentScores, List<String> stu
 
         return ranking;
     }
+
+    static int countOddScores(List<Integer> studentScores) {
+        int count = 0;
+
+        for (int score : studentScores) {
+            if (score % 2 == 0) {
+                continue;
+            }
+
+            count++;
+        }
+
+        return count;
+    }
+
+    static int evaluateChallengingExam(List<Integer> studentScores) {
+        int count = 0;
+
+        for (int score : studentScores) {
+            if (score <= 40) {
+                break;
+            }
+
+            count++;
+        }
+
+        return count;
```

Comment:
This actually makes for a nice `while` task:

```java
class MakingTheGrade {

    static int evaluateChallengingExam(List<Integer> studentScores) {
        int index = 0;

        while (studentScores.get(index) > 40) {
            index++;
        }

        return index + 1;
    }
}
```

😆

Reply:
Do you suggest to change this one for the one that is actually the while task? and make a new one for break ?

Reply:
@manumafe98, @kahgoh, happy to work with you on this one, guys.